src/back/spv/writer.rs

```diff
@@ -65,6 +65,7 @@ impl Writer {
             annotations: vec![],
             flags: options.flags,
             bounds_check_policies: options.bounds_check_policies,
+            zero_initialize_workgroup_memory: options.zero_initialize_workgroup_memory,
```

Comment:
thought(non-blocking): I feel like using pattern destructuring for data forwarding would be a really good fit for this sort of logic in `Self::new` and `Self::reset`. I'm the type that doesn't trust myself without that sort of safety net, tho, so YMMV. 😅

Reply:
I don't use destructuring much as I like seeing where the var came from at first glance.
@jimblandy thoughts?

Reply:
If I'm going to talk about things 'paying off', then I should say what the cost is: I think it takes a bit longer to read the destructuring version, there are more variables around that could be mixed with other things, and as Teo said, it's less obvious, when looking at a use, what its origin was.

That is, when I see `options.bounds_check_policies`, I know exactly where it came from, but when I see just `bounds_check_policies`, it could be an argument, it could have been computed, etc. and I have to guess or go find out.